docs/Transparency.md

```diff
@@ -50,29 +50,30 @@ Some assets were either transfered from CloudLinux, or have yet to be transferre
 
 The Board of Directors meets monthly on the second Tuesday of the month. At each meeting, the board works to keep minutes so that they can be approved as part of that meeting and shared immediately after the meeting completes. In the event that there is little to discuss, the Chair of the board may opt to cancel a meeting, allowing the board to meet every other month. 
 
-- June 14, 2023 [Board meeting #22 agenda](https://docs.google.com/document/d/1E1CTruoI-mGlSQ4Udb-ME4NKvtmSnK2sB1nC94a-APM/edit?usp=sharing)
-- May 10, 2023 [Board meeting #21 minutes](https://drive.google.com/file/d/17ZykiqjnYcr6OTs-pCuBP2oUc4wKQYye/view?usp=sharing)
+- July 12, 232 [Board meeting #23 agenda](https://docs.google.com/document/d/1qopFQH44WVykDpTTUwZAtb28DegGAKDXB5N2gT-WG6U/edit)
```

Comment:
I don't know if it matters, but this has a `/edit` in the URI which seems out of place.

Reply:
It does that because this isn’t a PDF L, but is a link to the living document.

Reply:
Ah, that makes sense. i didn't realize that was a future agenda.